Samples/blazor-add-in/excel-blazor-add-in/README.md

```diff
@@ -94,6 +94,25 @@ The fundamental pattern includes the following steps.
 1. Use **JSModule.InvokeVoidAsync** to call JavaScript functions from your C# code.
 1. Call Office JS APIs to interact with the worksheet from JavaScript code.
 
+## Debugging
+
+This sample is configured to support debugging both JavaScript and C# files. If you create a new Blazor project, you will need to make the following file updates to support C# debugging.
+
+1. In the **launchSettings.json** file of the web project, make sure all instances of `launchBrowser` are set to `false`.
+1. In the **<projectName>.csproj.user** file of the add-in project, add the `<BlazorAppUrl>` and `<InspectUri>` elements as shown in the following example XML.
+
+**Note:** The port number in the following XML is 7126. You must change the number to the port number specified in the **launchSettings.json** file for your web project.
```

Comment:
Maybe it's just me, but saying "it' here seems easier to understand than repeating "number".
```suggestion
**Note:** The port number in the following XML is 7126. You must change it to the port number specified in the **launchSettings.json** file for your web project.
```